### ISSUE TITLE
Bug: Focus ring / target size too small

### ISSUE THREAD

**Issue by bcullman** [SAP/fundamental#1277]
> Is this issue related to a specific component?
TreeView

> What version of the Fiori Fundamentals are you using?
1.4.2

**To Reproduce**
go to
https://sap.github.io/fundamental/components/tree.html

focus on one of the chevrons, and then use tab to gain focus on the close button.

**Screenshots**
![image](https://user-images.githubusercontent.com/5056972/52826782-22869e80-3077-11e9-96ce-7eb652679e86.png)

**Expected behavior**
target area (made visible with focus ring) is 44px in height and width, minimum a11y. 

to look nice, it should be square, filling the height of the entire row. a similar expand collapse chevron was used in SAP Concur's pre fundamentals library. here is where we landed:

![image](https://user-images.githubusercontent.com/5056972/52827104-45fe1900-3078-11e9-999d-e324a6352f22.png)
